src/platform/ESP32/CHIPDevicePlatformConfig.h

```diff
@@ -41,6 +41,15 @@
 #define CHIP_DEVICE_CONFIG_DEVICE_PRODUCT_ID CONFIG_DEVICE_PRODUCT_ID
 #define CHIP_DEVICE_CONFIG_DEFAULT_DEVICE_PRODUCT_REVISION CONFIG_DEFAULT_DEVICE_PRODUCT_REVISION
 #define CHIP_DEVICE_CONFIG_DEVICE_FIRMWARE_REVISION_STRING CONFIG_DEVICE_FIRMWARE_REVISION
+
+#if CONFIG_IDF_TARGET_ESP32H2
+#define CHIP_DEVICE_CONFIG_ENABLE_THREAD CONFIG_OPENTHREAD_ENABLED
+#define CHIP_DEVICE_CONFIG_ENABLE_THREAD_SRP_CLIENT CONFIG_OPENTHREAD_SRP_CLIENT
+#define CHIP_DEVICE_CONFIG_ENABLE_WIFI_AP 0
+#define CHIP_DEVICE_CONFIG_ENABLE_WIFI_STATION 0
+#define CHIP_DEVICE_CONFIG_ENABLE_WIFI_TELEMETRY 0
+#define CHIP_DEVICE_CONFIG_THREAD_ENABLE_CLI 0
```

Comment:
Should any of the above parameters be exposed through menuconfig? Or these values are never expected to change

Reply:
Removed these definitions